src/seedu/addressbook/AddressBook.java

```diff
@@ -816,6 +860,38 @@ private static void addPersonToAddressBook(HashMap<PersonProperty, String> perso
         ALL_PERSONS.add(person);
         savePersonsToFile(getAllPersonsInAddressBook(), storageFilePath);
     }
+    
+    /**
+     * Edits a person in the address book. Saves changes to storage file.
+     *
+     * @param updated person to edit
+     */
+    private static void editPersonToAddressBook(HashMap<PersonProperty, String> person) {
+    	if (updatePersonChanges(person)) {
+    		savePersonsToFile(getAllPersonsInAddressBook(), storageFilePath);
```

Comment:
Apologies, I already configured my eclipse to do that already! Its my personal habit of pressing the tab I enter a new line. Will take note of it!